addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -400,6 +412,12 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
             this.$target.removeClass('d-none');
             this.$message.addClass("d-none");
         }
+        const emailToInputEl = this.$target[0].querySelector('input[name="email_to"]');
+        if (emailToInputEl && !emailToInputEl.value && !this.dataForEmailTo) {
+            // If the user didn't change the Recipient Email, and there is no
+            // data-for value, we set the default value.
+            emailToInputEl.value = this.defaultEmailToValue;
+        }
```

Comment:
As explained above, either I am missing something or this is not a real use case. As soon as the user touch the form (adding it, choosing an action), the default value can be set up forcing the user to configure it IMO. The contact page is the only exception to me as there is a pre-existing form which works before entering edit mode.

Reply:
I agree, in reality, it doesn't matter, as the user is going to change the email recipient anyway for custom form. Since it's acceptable for you if the user saves a custom form without having touched the email recipient and in the options we have the default value but in the dom we have no email, I can remove some code in this PR :+1: